tests/integration/conftest.py

```diff
@@ -666,3 +666,32 @@ async def grafana_related_fixture(application: Application):
         raise_on_error=False,
     )
     return grafana
+
+
+@pytest_asyncio.fixture(scope="module", name="external_hostname")
+def external_hostname_fixture() -> str:
+    """Return the external hostname for ingress-related tests."""
+    return "juju.test"
+
+
+@pytest_asyncio.fixture(scope="module", name="ingress_related")
+async def ingress_related_fixture(application: Application, external_hostname: str):
```

Comment:
Would the fixture name be better as `ingress_application` rather than `ingress_related`? I've named `x_related` to denote the Jenkins application fixture that is related to the `x` application. Or I can change other fixture names to be clearer.